### ISSUE TITLE
Issues with published RPMs for SuSE 13.2

### ISSUE THREAD

**Issue by mikedougherty** [docker/docker#19944]
There seems to be some problems with the packages generated for SuSE 13.2 and hosted on yum.dockerproject.org. Here are the commands I've been testing with:

Start a SuSE container:
```bash
docker run --rm -ti --privileged opensuse:13.2 bash
```

Install docker as follows:
```
zypper --non-interactive install curl ca-certificates*
zypper -n ar -f obs://Virtualization:containers/SLE_12 Virtualization:containers
# Import Docker's key and the SuSE key
rpm --import 'https://sks-keyservers.net/pks/lookup?op=get&search=0x58118E89F3A912897C070ADBF76221572C52609D'
rpm --import 'https://sks-keyservers.net/pks/lookup?op=get&search=0x55A0B34D49501BB7CA474F5AA193FBB572174FC2'
curl -O https://yum.dockerproject.org/repo/main/opensuse/13.2/Packages/docker-engine-1.9.1-1.x86_64.rpm
zypper install docker-engine-1.9.1-1.x86_64.rpm
```
this will prompt you to:
1. ignore missing `libcgroup` dependency (choose option 2)
2. accept installation (choose option y)
Not sure where the libcgroup package is supposed to come from.

After install, note the output similar to:
```
Additional rpm output:
/var/tmp/rpm-tmp.wLf4tV: line 1: fg: no job control
```
This indicates that one of the RPM scripts failed; in this case I suspect a missing macro.

After all of that:
```
docker --version
Docker version 1.9.1, build %{_gitcommit}
```
Note the incorrect build identifier "%{_gitcommit}".

I'm not sure how to best debug further. 

/cc @jfrazelle @mavenugo

**Comment by thaJeztah** [comment]
ping @jfrazelle adding this to the milestone; guess we want this looked into/fixed before 1.10?

**Comment by jfrazelle** [comment]
Sure ya I mean I think it's always been like this clearly no ones using it
maybe we should just remove it 0:)

On Wednesday, February 3, 2016, Sebastiaan van Stijn <
notifications@github.com> wrote:

> ping @jfrazelle <https://github.com/jfrazelle> adding this to the
> milestone; guess we want this looked into/fixed before 1.10?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/issues/19944#issuecomment-179164226>.
>


-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>

**Comment by jfrazelle** [comment]
I fixed the libcgroup thing in the PR I made for 12.3 really if no one is
using this which it was already broken, then we shouldn't support it I can
fix it for CS who doesn't care about 13.1 anyways but I'm +1 to remove it
all together

On Wednesday, February 3, 2016, Jessica Frazelle <me@jessfraz.com> wrote:

> Sure ya I mean I think it's always been like this clearly no ones using it
> maybe we should just remove it 0:)
>
> On Wednesday, February 3, 2016, Sebastiaan van Stijn <
> notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> ping @jfrazelle <https://github.com/jfrazelle> adding this to the
>> milestone; guess we want this looked into/fixed before 1.10?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/docker/docker/issues/19944#issuecomment-179164226>.
>>
>
>
> --
>
>
> Jessie Frazelle
> 4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
> pgp.mit.edu
> <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>
>
>

-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>

**Comment by thaJeztah** [comment]
@jfrazelle perhaps we should remove it; our installation docs even tell users to use the SUSE repository https://docs.docker.com/engine/installation/SUSE/
